src/box/lua/net_box.lua

```diff
@@ -59,11 +60,23 @@ local IPROTO_FEATURE_NAMES = {
     [4]     = 'pagination',
 }
 
+local check_select_features = function(opts, remote)
+    if opts == nil or opts.after == nil and not opts.fetch_pos then
+        return
+    elseif not remote.peer_protocol_features.pagination then
+        return error("Pagination is not supported by remote server")
+    end
+end
+
+
+
 local REQUEST_OPTION_TYPES = {
     is_async    = "boolean",
     iterator    = "string",
     limit       = "number",
     offset      = "number",
+    fetch_pos   = "boolean",
+    after       = "any",
```

Comment:
`after` must be a string, a tuple, or an array, otherwise `netbox_encode_select` will fail with an assertion. Please check the parameter type here and add a test.

Reply:
Done